src/js/modules/supervisors/FileManager.ts

```diff
@@ -282,7 +295,11 @@ class FileManager {
             coprocessors
           );
           if (errors.find((errors) => errors.length > 0)) {
-            return Promise.reject(this.compactErrors(errors));
+            const compactedErrors = this.compactErrors(errors);
+            logger.error(
```

Comment:
You should remove `error` prefix from `src/js/modules/supervisors/HandleError.ts` in order to avoid duplicate `error` word on log message -> `2020-12-07T20:11:46.694Z [FileManager] error: enable_coprocessors RPC returned with some error: Error: error: topic "produce" doesn't exist`